src/crypto/CHIPCryptoPAL.h

```diff
@@ -628,6 +628,11 @@ class P256Keypair : public P256KeypairBase
     /** Release resources associated with this key pair */
     void Clear();
 
+#if defined(CHIP_CRYPTO_TRUSTY_OS)
```

Comment:
define checks should look like `#if CHIP_WHATEVER`.  That ensures that the define is actually defined when it should be, instead of silently doing odd things if the right headers or compile flags are missing.